docs/presentations/bmo-deep-dive/bmo-deep-dive.html

```diff
@@ -0,0 +1,409 @@
+	<!doctype html>
+	<html>
+		<head>
+			<meta charset="utf-8">
+			<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+
+			<title>reveal.js</title>
+
+			<link rel="stylesheet" href="dist/reset.css">
+			<link rel="stylesheet" href="dist/reveal.css">
+			<link rel="stylesheet" href="dist/theme/white.css" id="theme">
+
+
+			<!-- Theme used for syntax highlighted code -->
+			<link rel="stylesheet" href="plugin/highlight/monokai.css" id="highlight-theme">
+		</head>
+		<body>
+			<div class="reveal">
+				<div class="slides"
+				     style="font-size:50%;text-align:left">
+					<section data-markdown
+					         data-separator-vertical="^\n\n\n"
+									 data-background-image="https://github.com/metal3-io/metal3-docs/raw/master/images/metal3.png"
```

Comment:
after branch switch (master ==> main), we need to update this and all other BMO reference links in the document accordingly